src/gui/advancedsettings.cpp

```diff
@@ -863,6 +866,10 @@ void AdvancedSettings::loadAdvancedSettings()
     m_checkBoxMarkOfTheWeb.setChecked(pref->isMarkOfTheWebEnabled());
     addRow(ENABLE_MARK_OF_THE_WEB, motwLabel, &m_checkBoxMarkOfTheWeb);
 #endif // Q_OS_MACOS || Q_OS_WIN
+    // Validate TLS certificate
+    m_checkBoxValidateTLSCertificate.setChecked(pref->isValidateTLSCertificate());
+    m_checkBoxValidateTLSCertificate.setToolTip(tr("Validate TLS certificate for HTTPS URLs (e.g. RSS feeds, program updates, torrent files, geoip db, etc)"));
+    addRow(VALIDATE_TLS_CERTIFICATE, tr("Validate TLS certificates"), &m_checkBoxValidateTLSCertificate);
```

Comment:
Suggestions for better label and tooltip are welcome.

Reply:
Maybe just "Ignore SSL errors"?

>https://doc.qt.io/qt-6/qnetworkreply.html#ignoreSslErrors
>If this function is called, SSL errors related to network connection will be ignored, *including certificate validation errors.*

So it is doing more than just ignoring certificate errors.